### PR TITLE
FAQ on costs covered

### PR DESCRIPTION
**Pull request by semuelle**
Since this comes up quite often. Feel free to add or modify.

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -17,6 +17,12 @@
   Teams with a proven track record may apply for larger grants through [this application form](https://docs.google.com/forms/d/e/1FAIpQLSfMfjiRmDQDRk-4OhNASM6BAKii7rz_B1jWtbCPkUh6N7M2ww/viewform).
 </details>
 
+<details>
+  <summary><b>What activities/positions do you fund?</b></summary>
+
+  The Web3 Foundation's Grants Program aims to fund coding and research activities that are beneficial for the ecosystem as a whole. As such, we don't usually fund tangential costs, such as deployment and hosting costs, external consultants, maintenance or audits.
```
Comment (Noc2) [inline docs/faq.md:23]: For infrastruc we would fund audits and maintenance. Just not for projects that have a business model.

Comment (Noc2) [reply docs/faq.md:23]: And regarding external consultants, I guess it depends on what the external consultants actually do.

Comment (Noc2) [reply docs/faq.md:23]: But we don’t fund marketing or similar activities etc.

Comment (alxs) [reply docs/faq.md:23]: I thought we didn't fund infrastructure?

Comment (semuelle) [reply docs/faq.md:23]: > And regarding external consultants, I guess it depends on what the external consultants actually do.

Can you elaborate? Coders are fine, I suppose? "business-specific activities" perhaps?

Comment (semuelle) [reply docs/faq.md:23]: > For infrastruc we would fund audits and maintenance. Just not for projects that have a business model.

How can we phrase that? `not-for-profit projects`? `public good projects`?

Comment (Noc2) [reply docs/faq.md:23]: > I thought we didn't fund infrastructure?

We fund key infrastructure. I’m referring to api clients, the go implementation, etc.

Comment (Noc2) [reply docs/faq.md:23]: > > And regarding external consultants, I guess it depends on what the external consultants actually do.
> 
> Can you elaborate? Coders are fine, I suppose? "business-specific activities" perhaps?

Yes, I was referring to code is fine ;-)

Comment (Noc2) [reply docs/faq.md:23]: > > For infrastruc we would fund audits and maintenance. Just not for projects that have a business model.
> 
> How can we phrase that? `not-for-profit projects`? `public good projects`?

Sounds good.

File: docs/faq.md [hunk 2]
```diff
@@ -17,6 +17,12 @@
   Teams with a proven track record may apply for larger grants through [this application form](https://docs.google.com/forms/d/e/1FAIpQLSfMfjiRmDQDRk-4OhNASM6BAKii7rz_B1jWtbCPkUh6N7M2ww/viewform).
 </details>
 
+<details>
+  <summary><b>What activities/positions do you fund?</b></summary>
+
+  The Web3 Foundation's Grants Program aims to fund coding and research activities that are beneficial for the ecosystem as a whole. As such, we don't usually fund tangential costs, such as business-specific activities (marketing, business planning), events or outreach, and — for non-infrastructure projects — deployment and hosting costs, maintenance or audits. We also expect you to have a good understanding of the technologies you are planning to use, so we don't fund time spent learning how to use Substrate or how to write ink! smart contracts.
```
Comment (alxs) [inline docs/faq.md:23]:
```suggestion
  The Web3 Foundation's Grants Program aims to fund software development and research activities that are beneficial for the ecosystem as a whole. As such, we don't usually fund tangential costs such as business-oriented activities (marketing, business planning), events or outreach, and—for non-infrastructure projects—deployment and hosting costs, maintenance or audits. We also expect you to have a good understanding of the technologies you are planning to use, meaning that we don't fund time spent learning how to use Substrate or how to write ink! smart contracts.
```

**Review by Noc2 — APPROVED** [review]

**Review by alxs — APPROVED** [review]

**Review by mmagician — APPROVED** [review]

**Pull request merged**